### ISSUE TITLE
No CSS or documentation about it.

### ISSUE THREAD

**Issue by mechdog** [Globegitter/chartist-plugin-tooltip#12]
Is there any chance you could include and example .css for the tooltip style used on the Chartist site, or something along that lines? Or just post the css somewhere, it would cut the learning curve on your plug in by a lot. Think once that is sorted it is going to be AWESOME!

**Comment by anthony-dandrea** [comment]
Incase you haven't gotten the css sorted yet, here is my scss implementation:
```
$chart-tooltip-bg: #333333;
.chartist-tooltip {
    position: absolute;
    display: block;
    min-width: 5em;
    padding: .5em;
    background: $chart-tooltip-bg;
    color: #fff;
    font-weight: 400;
    text-align: center;
    pointer-events: none;
    z-index: 1;
    &:before {
        position: absolute;
        bottom: -14px;
        left: 50%;
        border: solid transparent;
        content: ' ';
        height: 0;
        width: 0;
        pointer-events: none;
        border-color: rgba(251, 249, 228, 0);
        border-top-color: $chart-tooltip-bg;
        border-width: 7px;
        margin-left: -8px;
    }
}
```

**Comment by mechdog** [comment]
Great thanks, I'll have to try that library out again now!!!

**Comment by shlomogoldstein** [comment]
Thanks man. The official CSS is not working at all.

**Comment by pdgehrke** [comment]
Using **bower install chartist-plugin-tooltip** installs v0.0.12 does not have the updated bowser.json file which includes the css path or the css files in dist:

This is the current bowser.json
"main": [
    "./dist/chartist-plugin-tooltip.min.js",
    "./dist/chartist-plugin-tooltip.css"
  ],

This is what installs with v0.0.12
"main": [
    "./dist/chartist-plugin-tooltip.min.js"
  ],

Can we get an updated current version? Thanks!

**Comment by ealves-pt** [comment]
Seems to be fixed with the release v0.0.13
